### PR TITLE
[[ Docs ]] Changes to precedence.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
* Escaped /* symbols so they would show up.
* Added more detail about operator precedence.

### CODE REVIEW

File: docs/glossary/p/precedence.lcdoc [hunk 1]
```diff
@@ -7,12 +7,16 @@ Type: glossary
...
-<operator>, so 2 + 3 * 4
-is <evaluate|evaluated> as 2 + (3 * 4), not (2 + 3) * 4.
+1. Operations grouped by parentheses.
+2. Exponentiation.
+3. Multiplication and division, from left to right.
+4. Addition and subtraction, from left to right.
+
+For example, the \* <operator> has higher precedence than the + 
+<operator>, so 2 + 3 \* 4
+is <evaluate|evaluated> as 2 + (3 \* 4), not (2 + 3) \* 4.
```
Comment (livecodeali) [inline docs/glossary/p/precedence.lcdoc:19]: For the first occurrence, does it work to actually link to the asterisk operator using `<*>`?

**Comment by asayd** [comment]
@livecodeali Using `<*>` didn't actually work, but I found a combination that did work. I still had to escape the \*.

**Comment by peter-b** [comment]
@asayd Thanks for this!

@livecode-vulcan review ok 3996b35

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 3996b35573330296e7e0968d323b655ae5397279

**Comment by livecode-vulcan** [comment]
😎 test success 3996b35573330296e7e0968d323b655ae5397279

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/709)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/710)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/707)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/708)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/714)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/705)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/706)

**Pull request merged**